README.md

```diff
@@ -28,6 +28,26 @@ Tracer.initGlobalTracer(LightStep.tracer({
 
 * **Node**: Node versions >= 0.12 are supported.
 
+## LightStep Initialization Options
+
+### Browser-specific
+
+#### `xhr_url_inclusion_patterns RegExp[]`
```

Comment:
(Just thinking out loud...)

This could also be something like `xhr_url_inclusion_patterns {operation_name, RegExp[]}`... i.e., each item in the whitelist could get an `operation_name`